pkg/base/config.go

```diff
@@ -213,6 +213,9 @@ type Config struct {
 	// connections to determine connection health and update the local view
 	// of remote clocks.
 	RPCHeartbeatInterval time.Duration
+
+	// Enables the use of an IEEE 1588 PTP clock device for HLC current time
+	ClockDevice        string
```

Comment:
This needs to be reset in `InitDefaults()` below.

Reply:
done